android/app/src/main/java/org/fossasia/openevent/data/Microlocation.java

```diff
@@ -24,69 +30,10 @@
     private int floor;
     private String room;
 
-    public Microlocation() {
-    }
+    public Microlocation() {}
```

Comment:
NoArgsConstructor should work even if there is a custom constructor, but if it's not working, then it's OK